common/presenters/events-to-timeline/event-details-helpers.test.js

```diff
@@ -0,0 +1,251 @@
+const proxyquire = require('proxyquire')
+
+const i18n = require('../../../config/i18n')
+
+const {
+  getMoveDetails,
+  getEventAgency,
+  getEventContext,
+  getEventRelationships,
+  getEventableDetails,
+  getEventProperties,
+} = proxyquire('./event-details-helpers', {
+  '../../../common/lib/api-client/models': {
+    event: {
+      fields: {
+        event_type: '',
+        prop_a: '',
+        prop_b: '',
+        details: '',
+        eventable: {
+          jsonApi: 'hasOne',
+        },
+        rel_a: {
+          jsonApi: 'hasOne',
+          type: 'rel_as',
+        },
+        rel_b: {
+          jsonApi: 'hasOne',
+          type: 'rel_bs',
+        },
+      },
+    },
+  },
+})
+
+describe('Timeline events', function () {
+  describe('Event details helpers', function () {
+    beforeEach(function () {
+      sinon.stub(i18n, 't').returnsArg(0)
+      sinon.stub(i18n, 'exists').returns(false)
+    })
+
+    context('#getMoveDetails', function () {
+      it('should return the move and person', function () {
+        const mockMove = { id: 'move', profile: { person: { id: 'person' } } }
+        const details = getMoveDetails(mockMove)
+        expect(details).to.deep.equal({
+          move: mockMove,
+          person: mockMove.profile.person,
+        })
+      })
+
+      it('should return just the move if it has no profile', function () {
+        const mockMove = { id: 'move' }
+        const details = getMoveDetails(mockMove)
+        expect(details).to.deep.equal({
+          move: mockMove,
+          person: undefined,
```

Comment:
Any thoughts on having a `person: undefined` vs the property not existing at all?

Reply:
i18n which is doing to use it doesn't care

Reply:
However, just in case something like `person.fullname` ends up somewhere that does care, I'm returning an empty object instead now